### DISCUSSION TITLE
Matlab for VirtualBox

### DISCUSSION THREAD

**Discussion by manueladn95** [brainvisa/brainvisa.github.io#114] [Q&A] [answered]
I am trying to install Matlab in the virtual box so I can set up a path to BrainVisa but I keep getting an error message. I downloaded it but when I attempt to run the installer that's when the error message comes up. I can't set up a path to Matlab that is already installed in my local machine because I'm using the virtual box. Any ideas on how can I get this resolved?
![Matlab error](https://user-images.githubusercontent.com/110187812/182692143-96f9db1f-3396-4825-9656-1ace6fdd7e5b.JPG)

**Comment by denisri** [comment, marked as answer]
Hi,
We're not experts of Matlab, so we may not be able to help you efficiently. Are you actually trying to install a Linux version of Matlab ? I'm asking because I see ".exe" which is usually a Windows executable extension. Just to be clear, the BrainVisa Virtualbox container is a Linux system, so you need to install Linux programs in it.
We usually don't use Matlab, but a standalone version of SPM, which doesn't require Matlab and a Matlab license, so we don't have experience in installing Matlab in the container.
Denis
